userdev/src/functionalTest/groovy/net/neoforged/gradle/userdev/CentralCacheTests.groovy

```diff
@@ -89,53 +91,6 @@ class CentralCacheTests extends BuilderBasedTestSpecification {
         cacheDir.listFiles().size() == 4
     }
 
-    def "cache_supports_running_gradle_in_parallel"() {
```

Comment:
I had to remove this test, because it is insanely flaky.
Not because of the central cache that it was testing that worked, but gradle itself simply not liking that it was running in parallel